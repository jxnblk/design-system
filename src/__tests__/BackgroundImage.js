import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { BackgroundImage, theme } from '..'

describe('BackgroundImage', () => {
  test('renders', () => {
    const json = renderer.create(<BackgroundImage theme={theme} />).toJSON()
    expect(json).toMatchSnapshot()
  })

  test('renders with image', () => {
    const json = renderer.create(<BackgroundImage image='hello.png' theme={theme} />).toJSON()
    expect(json).toMatchSnapshot()
  })

  test('renders with height', () => {
    const json = renderer.create(<BackgroundImage height='320px' theme={theme} />).toJSON()
    expect(json).toMatchSnapshot()
  })
})
