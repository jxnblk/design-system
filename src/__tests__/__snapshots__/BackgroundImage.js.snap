// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackgroundImage renders 1`] = `
.c0 {
  background-color: #596b7d;
  background-position: center;
  background-size: cover;
  background-repeat: no-repeat;
}

<div
  className="c0"
/>
`;

exports[`BackgroundImage renders with height 1`] = `
.c0 {
  height: 320px;
  background-color: #596b7d;
  background-position: center;
  background-size: cover;
  background-repeat: no-repeat;
}

<div
  className="c0"
  height="320px"
/>
`;

exports[`BackgroundImage renders with image 1`] = `
.c0 {
  background-image: url(hello.png);
  background-color: #596b7d;
  background-position: center;
  background-size: cover;
  background-repeat: no-repeat;
}

<div
  className="c0"
/>
`;
